ftdetect/set_repl_cmd.vim

```diff
@@ -10,8 +10,7 @@ if has('nvim') || has('terminal')
           \ end
     " Python
     au FileType python
-          \ let s:argList = split(g:neoterm_repl_python) |
-          \ if len(s:argList) > 0 && executable(s:argList[0]) |
+          \ if executable(g:neoterm_repl_python[-1]) |
```

Comment:
how will that behave when the value passed is a string?
Maybe we'll need something like
* `ftdetect/set_repl_cmd.vim`
```vimscript
if neoterm#repl#python#is_valid(g:neoterm_repl_python)
```
* `autoload/neoterm/repl/python.vim`
```vimscript
function! neoterm#repl#python#is_valid(value)
  if type(a:value) == v:t_list
  " ...
  else
  " ...
  end
endfunction
```

Reply:
Yes, I am already working on a solution here. However, the current solution seems to me a little too complex:

```
if type(g:neoterm_repl_pyhton) == v:t_list
  let l:pythonCmd = 0
  let l:invalidCmd = 0
  for i in range(len(g:neoterm_repl_python))
    if g:neoterm_repl_python[i] =~ 'python'
      let l:pythonCmd = 1
      if l:item == 'ipython'
        let g:neoterm_repl_python[i] = 'ipython --no-autoindent'
      endif
    endif
    if not executable(split(g:neoterm_repl_pyhton)[0])
      let l:invalidCmd = 1
    endif
endfor
else
" something similar for string
endif

if l:pythonCmd == 1 && l:invalidCmd == 0
    call neoterm#repl#set(g:neoterm_repl_python)
else
    echo "Invalid Command"
endif
```